release-notes/README.md

```diff
@@ -10,8 +10,22 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 * ACPCampaignClassic iOS SDK is now Adobe Campaign Classic (ACC) v8 compatible! BroadlogId can be provided in the UUID format in the notification tracking APIs.
 
+## May 12, 2022
+
+### iOS AEPAnalytics 3.1.0
+
+* Added support for using the Analytics Extension in [App Extensions](https://developer.apple.com/app-extensions/)
+
 ## May 9, 2022
 
+### iOS AEPCore 3.6.0
+
+* Added support for using the Core SDK in [App Extensions.](https://developer.apple.com/app-extensions/)
+* Added a new API to the Extension protocol for getting the latest non-pending shared state.
+* Added support for using Bundled Rules.
+* Added support for cached images for FullScreen Messages.
+* Fixed a bug preventing Fullscreen Messages from being dismissed in certain conditions.
```

Comment:
We've got "FullScreen" and "Fullscreen". Not sure which is correct, but we should make the capitalization consistent :) .

Reply:
I'm not sure where we're using FullScreen, but I think it should be Fullscreen.